change/@microsoft-teams-js-a84b5246-c410-443c-a67b-87d06e89f7c9.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "add timestamp to VideoFrame type, send timestamp after video frame processed",
```

Comment:
Our Contribution.md file says that change descriptions should be written in past tense and should surround code references in backticks.

Please update your comment to read:

"Added timestamp to `VideoFrame` type and return timestamp to the host after `VideoFrame` is processed"

Reply:
Changed the description. Thanks!